README.md

```diff
@@ -452,6 +452,17 @@ This example sends data to the EU site:
 
 On Windows, remove the `become: yes` option so the role does not fail. Below are two methods to make the example playbooks work with Windows hosts:
 
+To uninstall the Agent using Ansible, add the following code to your role:
+```
+- name: Check If Datadog Agent is installed
+  win_stat:
+  path: 'c:\Program Files\Datadog\Datadog Agent\bin\agent.exe'
+  register: stat_file
+- name: Remove windows datadog agent
+  win_shell: start-process msiexec -Wait -ArgumentList ('/log', 'C:\\uninst.log', '/norestart', '/q', '/x', (Get-WmiObject -Class Win32_Product -Filter "Name='Datadog Agent'" -ComputerName .).IdentifyingNumber)
+  when: stat_file.stat.exists == True
+```
+
```

Comment:
Could you move this to a new section below? The line above it refers to the next two sections, which should be placed immediately after it.